apps/emergency-call/index.html

```diff
@@ -60,8 +60,9 @@
           <button id="keypad-delete" class="reset-button" data-value="delete" data-l10n-id="keypadDelete">
           </button>
         </div>
-        <section id="ice-contacts-bar" class="ice-contacts-bar" hidden data-l10n-id="ice-contacts-bar-text">
-          ICE Contacts - In Case of Emergency
+        <section id="ice-contacts-bar" class="ice-contacts-bar" hidden>
+          <span class="placeholder"></span>
```

Comment:
The class name is a little too generic for my tastes. I'd rather have a dedicated id for this with a self-explanatory name such as `ice-contacts-bar-arrow`.